task/01-strings-tasks.js

```diff
@@ -55,7 +55,7 @@ function getStringLength(value) {
  *   'Chuck','Norris'  => 'Hello, Chuck Norris!'
  */
 function getStringFromTemplate(firstName, lastName) {
-    throw new Error('Not implemented');
+    return "Hello, " + `${firstName}` + " " + `${lastName}` + "!";
 }
```

Comment:
Why don't use template string for the whole expression?

Reply:
fix